azure-pipelines.yml

```diff
@@ -279,45 +329,56 @@ jobs:
         -DENABLE_DOWNLOADS=1
         -DDOWNLOAD_BOOST=1
         -DWITH_BOOST=$(BOOST_DIR)
-        -DMYSQL_MAINTAINER_MODE=ON
-        -DCMAKE_C_COMPILER_LAUNCHER=/usr/bin/ccache
```

Comment:
@inikep Not sure why exactly `ccache` is invoked after removing these lines.

Reply:
I solved the issue with Linux vs macOS using `-DCMAKE_C_COMPILER_LAUNCHER=$(which ccache)`.